documentation/src/main/asciidoc/ch02.asciidoc

```diff
@@ -494,6 +494,11 @@ impact portability of your application between Bean Validation providers.
 	Supported data types::: Any type
 	Hibernate metadata impact::: Column(s) are not nullable
 
+`@Negative(strict=)`:: Checks if the element is negative. If `strict=true` zero values are considered invalid, otherwise they are valid.
```

Comment:
@gunnarmorling I think docs are also missing Email and NotEmpty for BV constraints (didn't want to add them in this PR as they are not related to it, I can open another PR for them if needed)